server/devpi_server/views.py

```diff
@@ -239,6 +241,13 @@ def get_actions(json):
     return result
 
 
+@exception_view_config(ReadonlyIndex)
```

Comment:
I see no tests for the ``readonly_index_view``.

Reply:
See explanation for the exceptions.